src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,77 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int threshold = Math.round(DEFAULT_CAPACITY * LOAD_FACTOR);
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
```

Comment:
use the constructor to initialize this field
```suggestion
    private Node<K, V>[] table;
```